src/machine/usb/adc/midi/midi.go

```diff
@@ -79,7 +96,11 @@ func (m *midi) sendUSBPacket(b []byte) {
 }
 
 // from BulkIn
-func (m *midi) Handler() {
+func (m *midi) TxHandler() {
```

Comment:
Do we want to call this method `TxHandler`? And even more importantly- do we want to export it? What is there to gain?

Reply:
This is already the naming for transmit handler function in USB HID implementations elsewhere in TinyGo.